src/components/Tabber/Tabs/Tabs.module.scss

```
@@ -1,4 +1,5 @@
-﻿@use '../../../styles/global';
+@use '~styles/global';
+@use '~styles/breakpoints.scss';
```

```suggestion
@use '~styles/global' as *;
@use '~styles/breakpoints.scss' as *;
```
no octopost importamos os estilos assim, e na hora de usar as variaveis não precisa colocar `global.$variavel` só `$variavel` funciona

---

🤝